### ISSUE TITLE
Time-lapse images dashboard page

### ISSUE THREAD

**Issue by adamlewinter** [gadomski/glac.io#71]
Dave and I discussed this, but once the time-lapse viewer is complete and running for each camera site, we need to create a page dedicated to just the time-lapse imagery. This will display, in tiles, all of the most recent images from each camera. The user can then select a tile and go to the time-lapse viewer at each specific location.

Not sure if the link for this should be in the head, next to Locations, or possibly in a similar box as the "Monitoring Change" box on the homepage. 

In addition, we may want to do something similar for all of the climate station data. 

Not high priority, but wanted to get this down for future work.
